frappe/tests/test_boilerplate.py

```diff
@@ -55,39 +71,94 @@ def setUpClass(cls):
 			"public",
 		]
 
+	def create_app(self, hooks, no_git=False):
+		self.addCleanup(self.delete_test_app, hooks.app_name)
+		_create_app_boilerplate(self.apps_dir, hooks, no_git)
+
 	@classmethod
-	def tearDownClass(cls):
-		test_app_dirs = (os.path.join(cls.bench_path, "apps", app_name) for app_name in cls.app_names)
-		for test_app_dir in test_app_dirs:
-			if os.path.exists(test_app_dir):
-				shutil.rmtree(test_app_dir)
+	def delete_test_app(cls, app_name):
+		test_app_dir = os.path.join(cls.bench_path, "apps", app_name)
+		if os.path.exists(test_app_dir):
+			shutil.rmtree(test_app_dir)
+
+	@staticmethod
+	def get_user_input_stream(inputs):
+		user_inputs = []
+		for value in inputs.values():
+			if isinstance(value, list):
+				user_inputs.extend(value)
+			else:
+				user_inputs.append(value)
+		return StringIO("\n".join(user_inputs))
+
+	def test_simple_input_to_boilerplate(self):
+		with patch("sys.stdin", self.get_user_input_stream(self.default_user_input)):
```

Comment:
I've tried to patch input / use click test runner both are useless here. 

- Even tho input is used by click, since it's aliased it can't be patched.
- Click's test runner only gives you return value/output/side-effects, I just care about function's output. 

patching stdin is easiest way.